  main.cpp
c:\users\kevin\desktop\prog\myocontroller\myocontroller\myo\cxx\impl\../detail/ThrowOnError.hpp(39): warning C4297: 'myo::ThrowOnError::~ThrowOnError': function assumed not to throw an exception but does
  c:\users\kevin\desktop\prog\myocontroller\myocontroller\myo\cxx\impl\../detail/ThrowOnError.hpp(39): note: destructor or deallocator has a (possibly implicit) non-throwing exception specification
c:\users\kevin\desktop\prog\myocontroller\myocontroller\myo\cxx\impl\../detail/ThrowOnError.hpp(45): warning C4297: 'myo::ThrowOnError::~ThrowOnError': function assumed not to throw an exception but does
  c:\users\kevin\desktop\prog\myocontroller\myocontroller\myo\cxx\impl\../detail/ThrowOnError.hpp(45): note: destructor or deallocator has a (possibly implicit) non-throwing exception specification
c:\users\kevin\desktop\prog\myocontroller\myocontroller\Info.h(64): warning C4244: '=': conversion from 'double' to 'int', possible loss of data
c:\users\kevin\desktop\prog\myocontroller\myocontroller\Info.h(162): warning C4244: '=': conversion from 'double' to 'int', possible loss of data
c:\users\kevin\desktop\prog\myocontroller\myocontroller\ServerClient.h(128): warning C4267: 'initializing': conversion from 'size_t' to 'unsigned int', possible loss of data
  Generating code
  4 of 377 functions ( 1.1%) were compiled, the rest were copied from previous compilation.
    1 functions were new in current compilation
    6 functions had inline decision re-evaluated but remain unchanged
  Finished generating code
  MyoController.vcxproj -> C:\Users\kevin\Desktop\Prog\MyoController\x64\Release\MyoController.exe
  MyoController.vcxproj -> C:\Users\kevin\Desktop\Prog\MyoController\x64\Release\MyoController.pdb (Full PDB)
